app/src/main/res/values-ja/strings.xml

```diff
@@ -36,20 +36,20 @@
   <string name="property_weight">重さ</string>
   <string name="repository">リポジトリ</string>
   <string name="save">保存</string>
-  <string name="service_belpost">ベルポシュタ</string>
-  <string name="service_dpd_uk">DPD（イギリス）</string>
+  <string name="service_belpost">Belposhta (BY)</string>
```

Comment:
this is a transliteration from the native name of the delivery service. consider using Belpost (the official English name) or the native name.